cmd/pctl/search.go

```diff
@@ -25,41 +25,43 @@ func searchCmd() *cli.Command {
 				Value:       "table",
 				Usage:       "Output format. json|table",
 			},
+			&cli.BoolFlag{
+				Name:    "all",
+				Aliases: []string{"a"},
+				Usage:   "Search all available profiles",
+			},
 		},
 		Action: func(c *cli.Context) error {
-			searchName, catalogClient, err := parseArgs(c)
-			if err != nil {
-				_ = cli.ShowCommandHelp(c, "search")
-				return err
-			}
-
-			profiles, err := catalog.Search(catalogClient, searchName)
-			if err != nil {
-				return err
-			}
+			var profiles []profilesv1.ProfileCatalogEntry
 			outFormat := c.String("output")
```

Comment:
can we move this closer to where it is used?

Reply:
its used in the following if-else loop. i have moved the `outFormat` var down the line where its used